content/en/docs/instrumentation/net/metrics-api.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Metrics API reference
+linkTitle: .NET Metrics API
```

Comment:
To make these links more visually consistent with other language SIGs, how about this for a short title:

```suggestion
linkTitle: API - metrics
```